.eslintignore

```diff
@@ -1,4 +1,6 @@
 external
+node_modules
```

Comment:
That file and `.npmignore` feels weird to me, will evaluate them in the next pull